code/__DEFINES/traits/declarations.dm

```diff
@@ -688,6 +688,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_FOOD_SILVER "food_silver"
 /// If this item's been made by a chef instead of being map-spawned or admin-spawned or such
 #define TRAIT_FOOD_CHEF_MADE "food_made_by_chef"
+/// Added to sheets which are dispensed from a reagent holder
+#define TRAIT_SHEET_SMELTED "sheet_smelted"
```

Comment:
Optional: you could just make this into a boolean var on sheets rather than a trait.

Or maybe set gulag_valid = FALSE for smelted sheets and use that to validate the sheet for points